README.md

```diff
@@ -15,57 +15,66 @@ products:
 
 ![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)
 
-These sample projects showcase how to build Unity applications 
+These sample projects showcase how to build Unity applications
 for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
 For more details on installing related tools and setting up a Unity project,
 please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).
 
 > NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files,
-such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
+> such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
 
 > NOTE: Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime,
-which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
-To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
+> which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions).
+> To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
-Recommend to run these samples on HoloLens 2 using following version of tools:
+## Recommended tool versions
+
+It's recommended to run these samples on HoloLens 2 using the following versions:
 
 - Latest Visual Studio 2019
-- Unity 2020.3.6f1 LTS, or later
-- Unity OpenXR plugin 1.1.1, or later
-- Mixed Reality OpenXR plugin 0.9.3, or later
-- MRTK Unity 2.7.0-preview.4, or later
+- Unity 2020.3.8f1 LTS, or later
+- Unity OpenXR plugin 1.2.0, or later
+- Mixed Reality OpenXR plugin 0.9.4, or later
```

Comment:
Should this be 0.9.5?

Reply:
How about this?
- Latest Unity 2020.3 LTS, recommend 2020.3.6f1 or above.
- Latest Unity OpenXR plugin, recommend 1.2 or later
- Latest Mixed Reality OpenXR Plugin, recommend version 0.9.5 or later
- Latest MRTK, recommend version 2.7-preview.5 or later

Reply:
Good catch! Updated